scarb/build.rs

```diff
@@ -91,3 +92,20 @@ fn ident(id: &str) -> String {
     }
     ident
 }
+
+fn check_versions(comp_ver: &str, path: PathBuf) {
```

Comment:
```suggestion
fn check_corelib_version(core_scarb_toml: PathBuf) {
```

This function is `core`-specific anyway, so no need to overly generalise it